packages/pointer_interceptor/pointer_interceptor_ios/example/lib/main.dart

```diff
@@ -0,0 +1,68 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:pointer_interceptor/pointer_interceptor.dart';
```

Comment:
Ideally we don't want circular references between the package layers, so for the implementation packages what we'd want is to use `pointer_interceptor_platform_interrface` here, and `PointerInterceptorPlatform.instance.buildWidget` directly below.